api/controllers/UserController.js

```diff
@@ -63,6 +63,27 @@ module.exports = {
       }
       // Query for game if user is in one
       const gameId = (user.game ?? req.session.game) ?? null;
+
+      if (process.env.VITE_USE_GAMESTATE_API) {
+        const { unpackGamestate, createSocketEvent } = sails.helpers.gameStates;
+        const game = await Game.findOne(gameId).populate('gameStates');
+        const gameObject = await unpackGamestate(game.gameStates.at(-1));
+        const socketEvent = await createSocketEvent(game, gameObject);
+        
+        Game.subscribe(req, [ game.id ]);
+        req.session.loggedIn = true;
```

Comment:
Are we still using session.loggedIn? I thought we had switched to only reading session.usr instead